java/plantuml/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                plantuml
+version             1.2018.0
+revision            1
+categories          java editors
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
+maintainers         nomaintainer
+
+description         PlantUML is a tool for generating UML diagrams from textual \
+                    description
+
+long_description    ${description}
+
+homepage            https://sourceforge.net/projects/plantuml
+master_sites        sourceforge
+distfiles           plantuml-gplv2-${version}.tar.gz
+
+checksums           md5     167cfdb13ed557b3aae428084a6cd252 \
+                    sha1    fc89afc5728fe486c8239fb63e15389fc1370160 \
+                    rmd160  a454a43203294fb3f5a5236a997ef5f26fb811e1
+
+depends_build       bin:ant:apache-ant
+depends_lib         bin:java:kaffe
+
+extract.mkdir       yes
+
+use_configure       no
+
+build.cmd           ant
+build.target        
+
+destroot {
+    set javadir ${destroot}${prefix}/share/java
+
+    xinstall -d -m 755 ${javadir}
+
+    file copy ${worksrcpath}/${name}.jar ${javadir}/
```

Comment:
Shouldn't there be also some script in `${prefix}/bin/` to launch this? Only installing a `.jar` without any explanation is not that helpful.

Reply:
there is no script to run it. It is like java/ditaa.
You run it with "java -jar plantuml.jar".
I use it with emacs.

Reply:
I agree with @raimue, at the very least an explanation would be good, and possibly even a one line script?